src/decays/specializations/H/decay_H_to_AZ.inc

```diff
@@ -19,15 +19,27 @@ double CLASSNAME::get_partial_width<Higgs, Photon, ZBoson>(
       res *= alpha_em_0/alpha_em;
    }
 
+   std::string tag = field_as_string<Higgs>(in_idx) + "-" + field_as_string<Photon>(out1_idx) + "-" + field_as_string<ZBoson>(out2_idx);
    if (flexibledecay_settings.get(FlexibleDecay_settings::print_effc_block)) {
       effhiggscouplings_block_input.push_back(
          {
             25 + 10*(in_idx.size()>0 ? in_idx.at(0) : 0), 22, 23,
```

Comment:
See my comment from above: Please add a brief comment to explain this expression.

Reply:
A reference would also be ok.

Reply:
I have the feeling that my comment refers to the wrong line. It applies to the line
~~~.cpp
25 + 10*...
~~~

Reply:
This statement is no longer there.